src/components/NavBar/Bag.tsx

```diff
@@ -24,25 +25,24 @@ export const Bag = () => {
   const itemsInBag = useBag((state) => state.itemsInBag)
   const [bagQuantity, setBagQuantity] = useState(0)
 
+  const isNftProfilePage = useIsNftProfilePage()
+
   const { bagExpanded, setBagExpanded } = useBag(
     ({ bagExpanded, setBagExpanded }) => ({ bagExpanded, setBagExpanded }),
     shallow
   )
-  const { isSellMode, resetSellAssets, setIsSellMode } = useSellAsset(
-    ({ isSellMode, reset, setIsSellMode }) => ({
-      isSellMode,
+  const { resetSellAssets } = useSellAsset(
+    ({ reset }) => ({
       resetSellAssets: reset,
-      setIsSellMode,
     }),
     shallow
   )
   const handleIconClick = useCallback(() => {
-    if (isSellMode && bagExpanded) {
+    if (isNftProfilePage && bagExpanded) {
       resetSellAssets()
-      setIsSellMode(false)
     }
     setBagExpanded({ bagExpanded: !bagExpanded })
-  }, [bagExpanded, isSellMode, resetSellAssets, setBagExpanded, setIsSellMode])
+  }, [bagExpanded, isNftProfilePage, resetSellAssets, setBagExpanded])
```

Comment:
Hey,

I think you might not need this as I'm removing this behavior here

https://github.com/Uniswap/interface/pull/5224/files

Reply:
oh ok cool once you merge i will rebase this or close

Reply:
actually i think i can just close this entire pr